test/max.test.js

```diff
@@ -0,0 +1,8 @@
+import test from 'ava'
+import {max} from '../src'
```

Comment:
No, but it makes it easier for folks to use this package:

```js
import {max} from 'stack-overflow-copy-paste'
```

Even though nobody should really be using this package :wink: It's just for folks to practice contributing to OSS!